app/src/androidTest/java/com/github/sdp/ratemyepfl/activity/AddReviewActivityTest.kt

```diff
@@ -90,7 +107,7 @@ class AddRoomReviewActivityTest {
         onView(withId(R.id.roomReviewRatingBar)).perform(
             performSetRating(rating))
 
-        onView(withId(R.id.add_room_comment)).perform(typeText(comment))
+        onView(withId(R.id.add_review_comment)).perform(typeText(comment))
```

Comment:
Just a global remark that I thought about but did not mention in our last meeting: I think we should agree on the same naming convention for ids in XML files. According to what I read (here: https://infinum.com/handbook/android/building-quality-apps/naming), it seems that we should use camelCase in Kotlin (and snake_case in Java). However, the string resources are written in snake_case (I don't know why, but I'm sticking to this).

Reply:
We can stick to snake case I think, as long as everyone is the group does the same.

Reply:
I used the camelCase :'). But anyway it's about 5 min of refactoring (just press rename should not take too much time)

Reply:
Well I can do it in my other PR if needed, I was already cleaning all the code.